alkira/resource_alkira_connector_aruba_edge_test.go

```diff
@@ -29,32 +29,20 @@ func TestArubaEdgeDefalteInstance(t *testing.T) {
 	}
 }
 
-func TestDeflateArubaEdgeVrfMapping(t *testing.T) {
-	expectedVrfMapping := generateNumArubaEdgeVrfMapping(4)
-	m := deflateArubaEdgeVrfMapping(expectedVrfMapping)
-
-	for _, v := range m {
-		require.Contains(t, v, "advertise_on_prem_routes")
-		require.NotZero(t, v["advertise_on_prem_routes"])
-		require.Contains(t, v, "segment_id")
-		require.NotZero(t, v["segment_id"])
-		require.Contains(t, v, "aruba_edge_connect_segment_name")
-		require.NotZero(t, v["aruba_edge_connect_segment_name"])
-		require.Contains(t, v, "disable_internet_exit")
-		require.NotZero(t, v["disable_internet_exit"])
-		require.Contains(t, v, "gateway_gbp_asn")
-		require.NotZero(t, v["gateway_gbp_asn"])
-	}
-
-}
-
 func TestExpandArubaEdgeVrfMapping(t *testing.T) {
 	expectedArubaEdgeVrfMappings := generateNumArubaEdgeVrfMapping(3)
 
 	r := resourceAlkiraConnectorArubaEdge()
 	s := schema.NewSet(schema.HashResource(r), makeArrInterfaceFromArubaEdgeVrf(expectedArubaEdgeVrfMappings))
 
-	actualArubaEdgeVrfMappings, err := expandArubeEdgeVrfMapping(s)
+	getSegmentFn := func(id string) (alkira.Segment, error) {
+		i, err := strconv.Atoi(id)
+		require.Nil(t, err)
+
+		return alkira.Segment{Id: i, Name: id}, nil
+	}
+
+	actualArubaEdgeVrfMappings, err := expandArubaEdgeVrfMappings(s, getSegmentFn)
```

Comment:
I was thinking we will change this to `expandArubaEdgeVrfMappings(s, m)` instead?